proto/connect/v1/connect.proto

```diff
@@ -33,11 +35,10 @@ message SDKConnectRequestData {
 
 	string app_name = 3;
 	optional string environment = 4;
-	optional string framework = 5;
+	string framework = 5;
 	optional string platform = 6;
-	optional string sdk_version = 7;
-	optional string sdk_language = 8;
-	optional string sdk_author = 9;
+	string sdk_version = 7;
+	string sdk_language = 8;
```

Comment:
Should we add `SystemAttributes` to this type to add this data when establishing the connection?